veracruz-client/src/veracruz_client.rs

```diff
@@ -212,7 +234,7 @@ impl VeracruzClient {
             tls_session: session,
             remote_session_id: None,
             policy: policy,
-            policy_hash: policy_hash.to_string(),
+            policy_hash: policy_hash,
```

Comment:
Given the field and value's name both coincide you can omit the field name here.

Reply:
Though then it would be inconsistent with the other initializers in this statement